README.md

```diff
@@ -13,3 +13,23 @@ The *tutorials* folder contains walkthroughs which demonstrate basic Certifai fu
+| [HiddenFeatures.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/hidden_features) |  Illustrates the use of Certifai to analyze bias towards protected classes in a model. We show there can still be residual unfairness even if the model is not given access to the protected features. |  Binary classification |  Logistic Regression |  Fairness <br /> Explanations |  Protected features |
+| [PracticalIssues.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/practical_issues) - Practical Issues | Illustrates a couple of practical issues that may occur in typical deployments: <br /> - Pre-encoded data - this notebook shows how to handle datasets in which categorical features have already been encoded as one-hot columns (e.g. - by an ETL process). Explanations are still surfaced with normal categorical values, and the results will be equivalent to those which would have been obtained using an unencoded dataset (with a suitable encoder for the model). <br /> - Bucketing of granular or continuous fairness grouping features | Binary Classification | SVM <br /> Logistic Regression | Fairness (burden) | One-hot encoding <br /> Numerical fairness feature groups |
+| [ScanFromDefinition.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/scan_from_definition) - Running the scanner within a notebook from a definition file  | Illustrates the use of Certifai to generate reports based upon an existing scan definition, with multiple models and multiple analysis types which can be run as a single evaluation.   | Binary classification  | Logistic Regression  |  Fairness (by feature) |   |
+| [SHAP.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/shap) - SHAP explanations | Illustrates the use of an alternative explanation type (Kernel SHAP) to provide both an explainability measure for a model and explanations for individual predictions. SHAP may be used as an alternative to Certifai counterfactual explanation in circumstances where feature weights rather than counterfactual exemplars are preferred. | Binary Clasification | SVM <br /> Logistic Regression | Explanations  | SHAP Explanations |
+| [xgboostDmatrixExample.ipynb](https://github.com/CognitiveScale/cortex-certifai-examples/tree/master/notebooks/xgboost-model) - Running Cortex Certifai fairness evaluation on xgboost model to predict adult income | Illustrates how to accommodate Xgboost models trained with DMatrix data structure in Certifai Notebooks. | Binary Classification | Xboost (XBG) | Fairness | Xboost models <br /> DMatrix data structure |
```

Comment:
@sdraper-CS What do you think key properties should be for this example?

Reply:
I think maybe 'handling models with bespoke data encodings' or something?

Reply:
I changed the word "bespoke" since it's rather obscure to "suggested" so the key property is now: Handling models with suggested data encoding.

I also toyed with the words implied and inexplicit.